LocalDynamoDb/LocalDynamo.cs

```diff
@@ -56,10 +55,12 @@ public void Start()
         {
             Console.WriteLine("Starting in memory DynamoDb");
             var success = Dynamo.Start();
+            Thread.Sleep(2000);
```

Comment:
😢

Reply:
That's gone now, was only temporary whilst I was figuring some stuff out and I didn't want to async all the way up.